syft/frameworks/torch/tensors/interpreters/autograd.py

```diff
@@ -0,0 +1,139 @@
+from functools import wraps
+import torch
+
+import syft
+from syft.frameworks.torch.tensors.interpreters.abstract import AbstractTensor
+from syft.frameworks.torch.tensors.interpreters import PointerTensor
+from . import gradients
+
+
+def backwards_grad(grad_fn, in_grad=None):
+    back_grad = grad_fn(in_grad)
+    for next_grad_fn, next_grad in zip(grad_fn.next_functions, back_grad):
+        backwards_grad(next_grad_fn, next_grad)
+
+
+class AutogradTensor(AbstractTensor):
```

Comment:
The AutogradTensor seems to be missing a get() function. Could be something like:
```
    def get(self, deregister_ptr: bool = True):
        if self.grad is not None and hasattr(self.grad, 'child'):
            self.grad.get(deregister_ptr)

        self.child.get(deregister_ptr)
        return self
```

Reply:
I've added this in the last commits, thanks Silvia!